From 94d8f292dfe35d938ee9906b668412f2d757c78b Mon Sep 17 00:00:00 2001
From: Matt Flax <flatmax@flatmax.org>
Date: Tue, 4 Apr 2017 19:23:04 +1000
Subject: [PATCH 157/215] Audioinjector octo : Make the playback and capture
 symmetric

This patch ensures that the sample rate and channel count of the audioinjector
octo sound card are symmetric.
---
 sound/soc/bcm/audioinjector-octo-soundcard.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/bcm/audioinjector-octo-soundcard.c b/sound/soc/bcm/audioinjector-octo-soundcard.c
index 49115c8..5e79f4e 100644
--- a/sound/soc/bcm/audioinjector-octo-soundcard.c
+++ b/sound/soc/bcm/audioinjector-octo-soundcard.c
@@ -204,6 +204,8 @@ static int audioinjector_octo_trigger(struct snd_pcm_substream *substream,
 		.codec_dai_name = "cs42448",
 		.ops = &audioinjector_octo_ops,
 		.init = audioinjector_octo_dai_init,
+		.symmetric_rates = 1,
+		.symmetric_channels = 1,
 	},
 };
 
-- 
1.9.1

